client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -473,6 +479,9 @@ const mapStateToProps = state => {
 		hasSeoPreviewFeature: hasFeature( state, siteId, FEATURE_SEO_PREVIEW_TOOLS ),
 		isSaveSuccess: isSiteSettingsSaveSuccessful( state, siteId ),
 		saveError: getSiteSettingsSaveError( state, siteId ),
+		path: getCurrentRoute( state )
+			.replace( getSiteSlug( state, siteId ), ':site' )
+			.replace( siteId, ':siteid' ),
```

Comment:
I wonder if we shouldn't add this logic in a central place (such as [this one](https://github.com/Automattic/wp-calypso/blob/master/client/lib/analytics/utils.js)) since this is duplicating code from https://github.com/Automattic/wp-calypso/pull/32707. Plus we will probably need to perform such transformation in some other components in the future.

Reply:
I will refactor it in a follow up PR.